src/substrait/textplan/SymbolTablePrinter.cpp

```diff
@@ -141,15 +142,56 @@ std::string typeToText(const ::substrait::proto::Type& type) {
 std::string relationToText(
     const SymbolTable& symbolTable,
     const SymbolInfo& info) {
-  auto relation = ANY_CAST(const ::substrait::proto::Rel*, info.blob);
+  auto relationData = ANY_CAST(std::shared_ptr<RelationData>, info.blob);
 
   PlanPrinterVisitor printer(symbolTable);
-  return printer.printRelation(info.name, relation);
+  return printer.printRelation(info.name, relationData->protoAddr);
+}
+
+std::vector<std::string> followPipeline(
+    const SymbolTable& symbolTable,
+    const ::substrait::proto::Rel* relation) {
+  std::vector<std::string> pipeline;
+  auto info = symbolTable.lookupSymbolByLocation(
+      Location((google::protobuf::Message*)relation));
```

Comment:
Prefer static/dynamic cast over C style cast

Reply:
Weirdly enough static_cast wasn't working because of inheritance issues when in fact the problem was a missing const qualifier.  One of the other PRs has a convenience macro to hide this mass of text (and prevent errors)